backbone.js

```diff
@@ -613,43 +629,48 @@
 
       options = _.extend({validate: true, parse: true}, options);
       var wait = options.wait;
+      var success = options.success || _.identity;
+      var error = options.error;
+      delete options.success;
+      delete options.error;
```

Comment:
err, but the master branch does modify the `options` object already... I guess you object to the `delete` lines?